scripts/~Settings.nut

```diff
@@ -0,0 +1,7 @@
+local gt = this.getroottable();
+
```

Comment:
This file should be in /!config/ and begin with stronghold_ . In fact, this should be in stronghold_config, which should only include values that the average user will want to modify.

Reply:
i opted to keep it in root for sake of user friendliness, to avoid having to bug users to go to `scripts/!mod_preload/!config` to change stuff. 

can change if you insist.

Reply:
Good point, with a name change it is fine in root.

Reply:
after explaining why on discord, i namespaced it.